webapi/src/main/scala/org/knora/webapi/responders/v2/SearchResponderV2.scala

```diff
@@ -84,6 +84,9 @@ object SearchResponderV2Constants {
         val standoffPropVar: QueryVariable = QueryVariable("standoffProp")
         val standoffValueVar: QueryVariable = QueryVariable("standoffValue")
 
+        val listNode: QueryVariable = QueryVariable("listNode")
```

Comment:
What if the query involves two different list nodes? Shouldn't there be a unique variable name for each one?

I have the same question about the other instances of `QueryVariable` defined here.

Reply:
Consider the following query:

```
    PREFIX knora-api: <http://api.knora.org/ontology/knora-api/simple/v2#>  
    CONSTRUCT {
    
        ?mainRes knora-api:isMainResource true .
        
        ?mainRes <http://0.0.0.0:3333/ontology/0801/beol/simple/v2#hasSubject> ?propVal0 .
    
    } WHERE { 
    
        ?mainRes a knora-api:Resource .
        
        ?mainRes a <http://0.0.0.0:3333/ontology/0801/beol/simple/v2#letter> .
        
        
                    ?mainRes <http://0.0.0.0:3333/ontology/0801/beol/simple/v2#hasSubject> ?propVal0 .
                    <http://0.0.0.0:3333/ontology/0801/beol/simple/v2#hasSubject> knora-api:objectType <http://www.w3.org/2001/XMLSchema#string> .
                    ?propVal0 a <http://www.w3.org/2001/XMLSchema#string> .
                
            
            
        
    }
    
    OFFSET 0
```

the following main query is created from the given KnarQL:

```
 CONSTRUCT {
 ?mainResourceVar <http://www.knora.org/ontology/knora-base#isMainResource> "true"^^<http://www.w3.org/2001/XMLSchema#boolean> .
 ?mainAndDependentResource <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.knora.org/ontology/knora-base#Resource> .
 ?mainAndDependentResource ?mainAndDependentResourceProp ?mainAndDependentResourceObj .
 ?mainAndDependentResource <http://www.knora.org/ontology/knora-base#hasValue> ?mainAndDependentResourceValueObject .
 ?mainAndDependentResource ?mainAndDependentResourceValueProp ?mainAndDependentResourceValueObject .
 ?mainAndDependentResourceValueObject ?mainAndDependentResourceValueObjectProp ?mainAndDependentResourceValueObjectObj .
 ?mainAndDependentResourceValueObject <http://www.knora.org/ontology/knora-base#valueHasStandoff> ?standoffNode .
 ?standoffNode ?standoffProp ?standoffValue .
 ?mainAndDependentResourceValueObject <http://www.knora.org/ontology/knora-base#valueHasListNode> ?listNode .
 ?listNode <http://www.w3.org/2000/01/rdf-schema#label> ?listNodeLabel .
 } WHERE {
 {
 VALUES ?mainResourceVar { <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g> <http://rdfh.ch/0801/beol/1iT3P8KCQ9qSU4fWnwP3bg> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ> <http://rdfh.ch/0801/beol/2PeTcAP2R0uyhxKMR163pA> <http://rdfh.ch/0801/beol/1UyZAGwCRra25xK5f8pTSg> <http://rdfh.ch/0801/beol/-kF_VlCBQ6KTgUEYSAvfEg> <http://rdfh.ch/0801/beol/1ydw2I4nR1egWhm8OspBAg> <http://rdfh.ch/0801/beol/45jLFi29Rn-xl2wO8lCbkg> <http://rdfh.ch/0801/beol/1ODu_YNhQKyUwr2HPn___w> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA> <http://rdfh.ch/0801/beol/0EwJtpl-RVOANfhNE4lOdw> <http://rdfh.ch/0801/beol/0bUyZhemTt-Bdg93GJspjQ> <http://rdfh.ch/0801/beol/0Srw9xbhRuKwSqpGmwhMKw> <http://rdfh.ch/0801/beol/3hGjlHmGSEmv4xksG-u9Rg> <http://rdfh.ch/0801/beol/20dsBNORRzuqx2AolMH_Sg> <http://rdfh.ch/0801/beol/-y5jDcctRDy-6Ua9fe8fww> <http://rdfh.ch/0801/beol/2bihBFCwSrmbLK6kTTyALQ> <http://rdfh.ch/0801/beol/2kiTbDU-QSWIxN3tCcOKDg> <http://rdfh.ch/0801/beol/1vDaL57JQHyfjTydRAmW_A> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg> <http://rdfh.ch/0801/beol/05qZ_4zDRICkRoG5h4zoOQ> <http://rdfh.ch/0801/beol/0VuEzASaT7S8t7w191oZVg> <http://rdfh.ch/0801/beol/3tct2_3BRLmnSoA45OHBAA> <http://rdfh.ch/0801/beol/0_0Rs5IuT26hmcIGVu1NBA> <http://rdfh.ch/0801/beol/-0tI3HXgSSOeDtkf-SA00w> }
 ?mainResourceVar <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.knora.org/ontology/knora-base#Resource> .
 GRAPH <http://www.ontotext.com/explicit> {
     ?mainResourceVar <http://www.knora.org/ontology/knora-base#isDeleted> "false"^^<http://www.w3.org/2001/XMLSchema#boolean> .
 }
 } UNION {
 VALUES ?mainAndDependentResource { <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g> <http://rdfh.ch/0801/beol/1iT3P8KCQ9qSU4fWnwP3bg> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ> <http://rdfh.ch/0801/beol/2PeTcAP2R0uyhxKMR163pA> <http://rdfh.ch/0801/beol/1UyZAGwCRra25xK5f8pTSg> <http://rdfh.ch/0801/beol/-kF_VlCBQ6KTgUEYSAvfEg> <http://rdfh.ch/0801/beol/1ydw2I4nR1egWhm8OspBAg> <http://rdfh.ch/0801/beol/45jLFi29Rn-xl2wO8lCbkg> <http://rdfh.ch/0801/beol/1ODu_YNhQKyUwr2HPn___w> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA> <http://rdfh.ch/0801/beol/0EwJtpl-RVOANfhNE4lOdw> <http://rdfh.ch/0801/beol/0bUyZhemTt-Bdg93GJspjQ> <http://rdfh.ch/0801/beol/0Srw9xbhRuKwSqpGmwhMKw> <http://rdfh.ch/0801/beol/3hGjlHmGSEmv4xksG-u9Rg> <http://rdfh.ch/0801/beol/20dsBNORRzuqx2AolMH_Sg> <http://rdfh.ch/0801/beol/-y5jDcctRDy-6Ua9fe8fww> <http://rdfh.ch/0801/beol/2bihBFCwSrmbLK6kTTyALQ> <http://rdfh.ch/0801/beol/2kiTbDU-QSWIxN3tCcOKDg> <http://rdfh.ch/0801/beol/1vDaL57JQHyfjTydRAmW_A> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg> <http://rdfh.ch/0801/beol/05qZ_4zDRICkRoG5h4zoOQ> <http://rdfh.ch/0801/beol/0VuEzASaT7S8t7w191oZVg> <http://rdfh.ch/0801/beol/3tct2_3BRLmnSoA45OHBAA> <http://rdfh.ch/0801/beol/0_0Rs5IuT26hmcIGVu1NBA> <http://rdfh.ch/0801/beol/-0tI3HXgSSOeDtkf-SA00w> }
 ?mainAndDependentResource <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.knora.org/ontology/knora-base#Resource> .
 GRAPH <http://www.ontotext.com/explicit> {
     ?mainAndDependentResource <http://www.knora.org/ontology/knora-base#isDeleted> "false"^^<http://www.w3.org/2001/XMLSchema#boolean> .
 }
 GRAPH <http://www.ontotext.com/explicit> {
     ?mainAndDependentResource ?mainAndDependentResourceProp ?mainAndDependentResourceObj .
 }
 } UNION {
 VALUES ?mainAndDependentResourceValueObject { <http://rdfh.ch/0801/beol/1iT3P8KCQ9qSU4fWnwP3bg/values/MBpEl0J7S4C_erWObgzkUA> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/9GbjzXV0R_GmS36EjUT_fQ> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/u_K-he5BTcmKiRqOGWTVSg> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/_NazP9-mS52w0AQdH7D3mg> <http://rdfh.ch/0801/beol/0Srw9xbhRuKwSqpGmwhMKw/values/ml0tnFchRwqT87J6hc6R2A> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/y6VcU5D-Qzq8Tc2o-cAY7g> <http://rdfh.ch/0801/beol/0bUyZhemTt-Bdg93GJspjQ/values/hdK6ck0ORU6a5GouU4BqDQ> <http://rdfh.ch/0801/beol/-0tI3HXgSSOeDtkf-SA00w/values/rfao0tqnQTiYyLdlJrNF6w> <http://rdfh.ch/0801/beol/05qZ_4zDRICkRoG5h4zoOQ/values/uN-kDHqHRxC-AH_jpPcIkQ> <http://rdfh.ch/0801/beol/-y5jDcctRDy-6Ua9fe8fww/values/6-WLuWHIStm3UHUso9hS8Q> <http://rdfh.ch/0801/beol/3hGjlHmGSEmv4xksG-u9Rg/values/fpX5xC8dRVi9viOiJW1Q8A> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/asuHddlHT5W6rHBirMGRCA> <http://rdfh.ch/0801/beol/-kF_VlCBQ6KTgUEYSAvfEg/values/28Mo6KbWRemI93GB0pgw5g> <http://rdfh.ch/0801/beol/3hGjlHmGSEmv4xksG-u9Rg/values/zq_D6-B4TF-mhYTT125Qig> <http://rdfh.ch/0801/beol/-y5jDcctRDy-6Ua9fe8fww/values/TxRLqvY0Sd2C_3cKuMVhNA> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/JtQC1WszRzy3qFdJhHqJ3A> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/7N_gmWSESxmCdsmGQonjUA> <http://rdfh.ch/0801/beol/0bUyZhemTt-Bdg93GJspjQ/values/WnPMqisITEixK-k_1sYYIg> <http://rdfh.ch/0801/beol/0EwJtpl-RVOANfhNE4lOdw/values/CJDRE1uNSdqS93XYE2uPUQ> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/MTA2NLkARHGnL49poTnnCw> <http://rdfh.ch/0801/beol/0VuEzASaT7S8t7w191oZVg/values/dKj_p6N_TryhpzlvOSDzCQ> <http://rdfh.ch/0801/beol/2kiTbDU-QSWIxN3tCcOKDg/values/0n6Zs4eER96-nKpWsqEOdQ> <http://rdfh.ch/0801/beol/1iT3P8KCQ9qSU4fWnwP3bg/values/d5SALRuOTR2FRebOGOA7eA> <http://rdfh.ch/0801/beol/2kiTbDU-QSWIxN3tCcOKDg/values/VkCF3KX0SqCZorIWjR7sbw> <http://rdfh.ch/0801/beol/1iT3P8KCQ9qSU4fWnwP3bg/values/F0bAh82OT6GjYwjt4QhOiQ> <http://rdfh.ch/0801/beol/2PeTcAP2R0uyhxKMR163pA/values/A7jizu3WReqfBsH5-lr4rw> <http://rdfh.ch/0801/beol/45jLFi29Rn-xl2wO8lCbkg/values/VQnWaOOVRYyxjfYrH5r0Ng> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/KQSKsJ96Ro6h2hKj9Bg7rA> <http://rdfh.ch/0801/beol/2bihBFCwSrmbLK6kTTyALQ/values/4GESJftCTtCHumi1lr1tRA> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/x230cFHYSHmE-W7_vXSmzw> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/ywenGo5uQLqG6Ap1EhWTqA> <http://rdfh.ch/0801/beol/-y5jDcctRDy-6Ua9fe8fww/values/B4KX2K8MTBmtfUwv4gu98A> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/1ozllxMJR3aSOcXBHWGWHg> <http://rdfh.ch/0801/beol/1ODu_YNhQKyUwr2HPn___w/values/Uv5hRdgtTJCmRWwlvAD-xA> <http://rdfh.ch/0801/beol/2bihBFCwSrmbLK6kTTyALQ/values/HHgYbn0qTXG9mu9uFpaj6w> <http://rdfh.ch/0801/beol/3hGjlHmGSEmv4xksG-u9Rg/values/0Hoczfn_RJCO4Ts7l8khNQ> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/XcDyt3prSZmBLU_HxKuQZQ> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/vCHSapKDTgGsv0M9Q8y8dA> <http://rdfh.ch/0801/beol/3tct2_3BRLmnSoA45OHBAA/values/vJrRiaGNSW6RgmIo2Br5JQ> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/OWX4R7SzRmqInZWbxlHo_A> <http://rdfh.ch/0801/beol/45jLFi29Rn-xl2wO8lCbkg/values/JjNZgeLaQPG0_nlHAOPa8g> <http://rdfh.ch/0801/beol/2bihBFCwSrmbLK6kTTyALQ/values/KYDm7E75SaSvbMZst4kLUA> <http://rdfh.ch/0801/beol/2kiTbDU-QSWIxN3tCcOKDg/values/6244B--ZTJGpOvGtBtpWGQ> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/10N5zTK_SLW_DUbOCHcF9w> <http://rdfh.ch/0801/beol/3tct2_3BRLmnSoA45OHBAA/values/mqMgJJ-aSj2ODSMbVuT5UA> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/_bZoz-M5T5uHmiIaHWZSow> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/px20BpBCRVC_8RAUzDDrdg> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/NqNyUSqSS2u27iMHTg-HjA> <http://rdfh.ch/0801/beol/2kiTbDU-QSWIxN3tCcOKDg/values/d0kx2cweRFWNKSs3uC7Wzw> <http://rdfh.ch/0801/beol/2bihBFCwSrmbLK6kTTyALQ/values/TZlz-8DUQxet7BTfuunYJA> <http://rdfh.ch/0801/beol/1UyZAGwCRra25xK5f8pTSg/values/kARq28WyTQuNS0dA2FEkTg> <http://rdfh.ch/0801/beol/-kF_VlCBQ6KTgUEYSAvfEg/values/5k7MFQjtQ6e3Zt2TVkl9HA> <http://rdfh.ch/0801/beol/05qZ_4zDRICkRoG5h4zoOQ/values/Gv3TBnvATTqlc6JKR-jQkA> <http://rdfh.ch/0801/beol/1ydw2I4nR1egWhm8OspBAg/values/f_LmajcBSmaxSzueLcPgjA> <http://rdfh.ch/0801/beol/1ydw2I4nR1egWhm8OspBAg/values/mOQJ0-6eQuOQLsC2akOOpg> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/Ijxfft75Qaau9u99_mMRhg> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/VShDWtA8TiKWqQHdd0qpMA> <http://rdfh.ch/0801/beol/45jLFi29Rn-xl2wO8lCbkg/values/32TcQXzVRT6AsUN1yxhhcQ> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/yhTgFLicSPegW1Y9eNRjXw> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/n0k5mgNVQPSi23RnYUmaUw> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/VefelQ2KTjmbHMTTKTD6sg> <http://rdfh.ch/0801/beol/2PeTcAP2R0uyhxKMR163pA/values/Mx_RUJYQQ-SEaB86g0Rp6g> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/ZCXeZVIYQZ-AoAk-43eMDw> <http://rdfh.ch/0801/beol/05qZ_4zDRICkRoG5h4zoOQ/values/h6So-5zhTDKyJV02dUAVSg> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/Bw1Dk4GwRwa9jvl50Wwq4Q> <http://rdfh.ch/0801/beol/-kF_VlCBQ6KTgUEYSAvfEg/values/q6KbEhHTQ_i8NrQBR3Q-YA> <http://rdfh.ch/0801/beol/0EwJtpl-RVOANfhNE4lOdw/values/Uzmrc-G1RX2qS8JK6H3TOg> <http://rdfh.ch/0801/beol/1UyZAGwCRra25xK5f8pTSg/values/PVhrOO4vT8un3Dr8pP2VvA> <http://rdfh.ch/0801/beol/0EwJtpl-RVOANfhNE4lOdw/values/aLy4snJZT16aLhDoLZfLRQ> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/9XS3TKbYSAGhxHHYnudssA> <http://rdfh.ch/0801/beol/2PeTcAP2R0uyhxKMR163pA/values/H3HT9cH_RE-g86jibxqdUQ> <http://rdfh.ch/0801/beol/2kiTbDU-QSWIxN3tCcOKDg/values/VCP6S5yMRAKlN3z4o0SEDg> <http://rdfh.ch/0801/beol/0_0Rs5IuT26hmcIGVu1NBA/values/7irnXvVBSA6B4SdXOY-7ow> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/pgQkZm1aTcikAcI5HYtERg> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/BqX2y-caRS2WXpWa4_IPYQ> <http://rdfh.ch/0801/beol/3tct2_3BRLmnSoA45OHBAA/values/io8YbHN_RuCNOZSXEreeMQ> <http://rdfh.ch/0801/beol/-0tI3HXgSSOeDtkf-SA00w/values/H4IZDQ1eS2WBrMhZRX-vkA> <http://rdfh.ch/0801/beol/1UyZAGwCRra25xK5f8pTSg/values/fvamE-RkS5i_f840TikfSA> <http://rdfh.ch/0801/beol/45jLFi29Rn-xl2wO8lCbkg/values/rnbrpUM2Q1iaSuNmCnmuLw> <http://rdfh.ch/0801/beol/20dsBNORRzuqx2AolMH_Sg/values/Njdq3nD4SxuaY7k3w63DDA> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/-pEgTyERReG206isJEJtLQ> <http://rdfh.ch/0801/beol/2bihBFCwSrmbLK6kTTyALQ/values/FCWcd4NiQX2EaSPm-XQMgw> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/x5WXFx6fTdSJtoa5MH7H6Q> <http://rdfh.ch/0801/beol/3tct2_3BRLmnSoA45OHBAA/values/Rz9LOthjQ7-U3IG41UkGeA> <http://rdfh.ch/0801/beol/2PeTcAP2R0uyhxKMR163pA/values/tNwzPY_cRpOgl-1fwvPezQ> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/lktHppa5RqyPmwVFRUBP4Q> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/Wkprd358Rma9REl0jKtToA> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/doxdoy6rSqOXIxJ6te3bXw> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/kW7c3dw-Rj6mb2sKytkYPQ> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/SJRySI6MTraQCMW7vTooOw> <http://rdfh.ch/0801/beol/0EwJtpl-RVOANfhNE4lOdw/values/iKg-VFviQdKT7xRaP72axQ> <http://rdfh.ch/0801/beol/-0tI3HXgSSOeDtkf-SA00w/values/MdjXlOTNRfqYKEWNIh6kfg> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/PKUrOieLSBysE2aXgPpA5Q> <http://rdfh.ch/0801/beol/2bihBFCwSrmbLK6kTTyALQ/values/dRdJh8djSZyDLcAkWA6Nsg> <http://rdfh.ch/0801/beol/2kiTbDU-QSWIxN3tCcOKDg/values/f7NhXm0eS9uyWZrTF4md5g> <http://rdfh.ch/0801/beol/-0tI3HXgSSOeDtkf-SA00w/values/SiEBL-EASQSNeFBprbjY3A> <http://rdfh.ch/0801/beol/-kF_VlCBQ6KTgUEYSAvfEg/values/iLteGtvsSOeXL_bZDNoVdw> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/LKuE2smyQa6QJ-kzUjEv-A> <http://rdfh.ch/0801/beol/0Srw9xbhRuKwSqpGmwhMKw/values/CK9iBb_WRO2YB1qYxbQo_Q> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/taW-kxP_RQ2Phw_4eJ_heg> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/wjAIl6kITD2cgiK5CnPuSA> <http://rdfh.ch/0801/beol/2bihBFCwSrmbLK6kTTyALQ/values/0fnxjaMdTjGXGI7CL-NtJA> <http://rdfh.ch/0801/beol/0_0Rs5IuT26hmcIGVu1NBA/values/FsfjsypWTL-qRz9lyvbUSA> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/5b_NqIM8RGS0-zvCGDIk7A> <http://rdfh.ch/0801/beol/-0tI3HXgSSOeDtkf-SA00w/values/JpGklfqZSxuu7VI1zIyucw> <http://rdfh.ch/0801/beol/2bihBFCwSrmbLK6kTTyALQ/values/Kd9hDnpXRi6l14jEFrTSiQ> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/pJxNtVEtSB-zbhPp4Aom_w> <http://rdfh.ch/0801/beol/20dsBNORRzuqx2AolMH_Sg/values/-oZMShYwSxy2ewacvHqZbQ> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/O8ZKgJMOTX-lugiCMil4_g> <http://rdfh.ch/0801/beol/45jLFi29Rn-xl2wO8lCbkg/values/oiEAAaXoTryPlr2se50Wcg> <http://rdfh.ch/0801/beol/45jLFi29Rn-xl2wO8lCbkg/values/skEd2S6ZRRee5Qik7EL4Og> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/9-lATMDJQqOdTu2odTZZUw> <http://rdfh.ch/0801/beol/1vDaL57JQHyfjTydRAmW_A/values/5OEnzIsXRFWWlgR5bkYX2w> <http://rdfh.ch/0801/beol/-y5jDcctRDy-6Ua9fe8fww/values/xoDD_qMWTVOl6Mbdbishzw> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/-ZSP4wHLS-GvgoQDscRrZg> <http://rdfh.ch/0801/beol/-kF_VlCBQ6KTgUEYSAvfEg/values/HSf2Lf2jTtGelR4zy3LSIg> <http://rdfh.ch/0801/beol/2PeTcAP2R0uyhxKMR163pA/values/bU959THbQmS5dz-P8YvYgA> <http://rdfh.ch/0801/beol/1iT3P8KCQ9qSU4fWnwP3bg/values/slcn-zU5RsO0Pu3a5VdE1A> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/utmWuFHwRSOZZ4R26ncJxA> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/0BQZFWOcTNuH3oqaPNtTqw> <http://rdfh.ch/0801/beol/0_0Rs5IuT26hmcIGVu1NBA/values/fE2zQTBHS_O7yH08FBjuQQ> <http://rdfh.ch/0801/beol/-y5jDcctRDy-6Ua9fe8fww/values/RMJirK-dR8CqBQavcz7lEw> <http://rdfh.ch/0801/beol/0_0Rs5IuT26hmcIGVu1NBA/values/SCzr4s99SgiZPAKePj-uaw> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/ERs4CNcjQRufYDeuyn9PjA> <http://rdfh.ch/0801/beol/-kF_VlCBQ6KTgUEYSAvfEg/values/0XnDPrYcQjqtUnOHmpkFCg> <http://rdfh.ch/0801/beol/2PeTcAP2R0uyhxKMR163pA/values/TVJgVanJRS2bR6ZJ6GOzMw> <http://rdfh.ch/0801/beol/0_0Rs5IuT26hmcIGVu1NBA/values/OnaYkzmKTXmEygrtWIDsww> <http://rdfh.ch/0801/beol/05qZ_4zDRICkRoG5h4zoOQ/values/PgYgjxi6TG6QW90B4EJqZg> <http://rdfh.ch/0801/beol/1iT3P8KCQ9qSU4fWnwP3bg/values/2EfIxCTVSEG73hsJYdXmww> <http://rdfh.ch/0801/beol/0_0Rs5IuT26hmcIGVu1NBA/values/siEPVzsERLajHyn8x_vkrA> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/4TXvKs5kRnmXM2Jw06w00g> <http://rdfh.ch/0801/beol/0_0Rs5IuT26hmcIGVu1NBA/values/6JLefDWsS9aG1YOgHzD6yg> <http://rdfh.ch/0801/beol/05qZ_4zDRICkRoG5h4zoOQ/values/Q1nnqbguR-qgpqxTvaLCSw> <http://rdfh.ch/0801/beol/1UyZAGwCRra25xK5f8pTSg/values/aM1Ap-G2SqCckpBX56D_rA> <http://rdfh.ch/0801/beol/20dsBNORRzuqx2AolMH_Sg/values/qtJblLZXTqu65_-Z1kxCuQ> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/7DLjc0EbTfWR27FslclL_A> <http://rdfh.ch/0801/beol/-kF_VlCBQ6KTgUEYSAvfEg/values/a6HVX11wScyYfPX8-6ZCOQ> <http://rdfh.ch/0801/beol/1ODu_YNhQKyUwr2HPn___w/values/UimY2UwOTPOGUIAbSgWwNQ> <http://rdfh.ch/0801/beol/45jLFi29Rn-xl2wO8lCbkg/values/reOkePFBT_2SIgwXYm11Sg> <http://rdfh.ch/0801/beol/0bUyZhemTt-Bdg93GJspjQ/values/RsxHJaTYSwC4p-_iYlQzEQ> <http://rdfh.ch/0801/beol/-y5jDcctRDy-6Ua9fe8fww/values/_35SrJ06Tj2A0DGrp3K1_g> <http://rdfh.ch/0801/beol/0EwJtpl-RVOANfhNE4lOdw/values/_SoSJvp2SIWbuOdFAbJhxQ> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/bCoNkiVhR5askrEdd2ARLA> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/giKsWTNHSl2_38AsiGyGcA> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/S-ZxGoaiRoeh-AoObij4QQ> <http://rdfh.ch/0801/beol/45jLFi29Rn-xl2wO8lCbkg/values/ROgm43QaSdCKJ6iyaYizyA> <http://rdfh.ch/0801/beol/2PeTcAP2R0uyhxKMR163pA/values/wpu1FsIDTHCvda-3r0yEqg> <http://rdfh.ch/0801/beol/-y5jDcctRDy-6Ua9fe8fww/values/1Rllte5ISbKUbjkxYivIYw> <http://rdfh.ch/0801/beol/-y5jDcctRDy-6Ua9fe8fww/values/huHXJNZGRZm5xPOPb1Z0WA> <http://rdfh.ch/0801/beol/0EwJtpl-RVOANfhNE4lOdw/values/gVr85VTmRz6t9de0fFRi2Q> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/fLtq-DidTcyqPJbOWGzz-w> <http://rdfh.ch/0801/beol/1ODu_YNhQKyUwr2HPn___w/values/V7xg7qIsRDmPe9n9L-i7vg> <http://rdfh.ch/0801/beol/-y5jDcctRDy-6Ua9fe8fww/values/pHt71EfiQWmBoQdW4AONQQ> <http://rdfh.ch/0801/beol/2bihBFCwSrmbLK6kTTyALQ/values/WcGtZKi1Qh2Io1CdBrdGww> <http://rdfh.ch/0801/beol/05qZ_4zDRICkRoG5h4zoOQ/values/DayNSrv7R6an8HZcLZA-Aw> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/yIO7rcfRQy6xxRfGQBETag> <http://rdfh.ch/0801/beol/0Srw9xbhRuKwSqpGmwhMKw/values/CidpHDYUS0KPIX7ctB47ig> <http://rdfh.ch/0801/beol/1vDaL57JQHyfjTydRAmW_A/values/dMxC6iXHSLmtHaxbebgZig> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/0eJYDFGhQKubCsT4syxHew> <http://rdfh.ch/0801/beol/0Srw9xbhRuKwSqpGmwhMKw/values/fySgOSCbR8GSogPoS6sdwA> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/OhDJUFahRXmz-Opm2GdOqA> <http://rdfh.ch/0801/beol/0_0Rs5IuT26hmcIGVu1NBA/values/NmDVv68YQwCC4O8LCX5HUw> <http://rdfh.ch/0801/beol/45jLFi29Rn-xl2wO8lCbkg/values/nh1uuWQlRZG11qDxzvJZSA> <http://rdfh.ch/0801/beol/-kF_VlCBQ6KTgUEYSAvfEg/values/sRwUmIRwRUaOiaajRqaqBQ> <http://rdfh.ch/0801/beol/-0tI3HXgSSOeDtkf-SA00w/values/_fCIiKjzRYqjXkEhykto5A> <http://rdfh.ch/0801/beol/1UyZAGwCRra25xK5f8pTSg/values/Z0pXX3I9S86ZZwTrd1QDgA> <http://rdfh.ch/0801/beol/3tct2_3BRLmnSoA45OHBAA/values/tp3PjHLaSj6BE-sYe3eLxQ> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/TapV8cKAR4KdESbbS6LO9Q> <http://rdfh.ch/0801/beol/3tct2_3BRLmnSoA45OHBAA/values/TAFDaJEtTj2jsi6zFvOGkA> }
 ?mainAndDependentResource <http://www.knora.org/ontology/knora-base#hasValue> ?mainAndDependentResourceValueObject .
 GRAPH <http://www.ontotext.com/explicit> {
     ?mainAndDependentResource ?mainAndDependentResourceValueProp ?mainAndDependentResourceValueObject .
 }
 GRAPH <http://www.ontotext.com/explicit> {
     ?mainAndDependentResourceValueObject <http://www.knora.org/ontology/knora-base#isDeleted> "false"^^<http://www.w3.org/2001/XMLSchema#boolean> .
 }
 GRAPH <http://www.ontotext.com/explicit> {
     ?mainAndDependentResourceValueObject ?mainAndDependentResourceValueObjectProp ?mainAndDependentResourceValueObjectObj .
 }
 } UNION {
 VALUES ?mainAndDependentResourceValueObject { <http://rdfh.ch/0801/beol/1iT3P8KCQ9qSU4fWnwP3bg/values/MBpEl0J7S4C_erWObgzkUA> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/9GbjzXV0R_GmS36EjUT_fQ> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/u_K-he5BTcmKiRqOGWTVSg> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/_NazP9-mS52w0AQdH7D3mg> <http://rdfh.ch/0801/beol/0Srw9xbhRuKwSqpGmwhMKw/values/ml0tnFchRwqT87J6hc6R2A> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/y6VcU5D-Qzq8Tc2o-cAY7g> <http://rdfh.ch/0801/beol/0bUyZhemTt-Bdg93GJspjQ/values/hdK6ck0ORU6a5GouU4BqDQ> <http://rdfh.ch/0801/beol/-0tI3HXgSSOeDtkf-SA00w/values/rfao0tqnQTiYyLdlJrNF6w> <http://rdfh.ch/0801/beol/05qZ_4zDRICkRoG5h4zoOQ/values/uN-kDHqHRxC-AH_jpPcIkQ> <http://rdfh.ch/0801/beol/-y5jDcctRDy-6Ua9fe8fww/values/6-WLuWHIStm3UHUso9hS8Q> <http://rdfh.ch/0801/beol/3hGjlHmGSEmv4xksG-u9Rg/values/fpX5xC8dRVi9viOiJW1Q8A> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/asuHddlHT5W6rHBirMGRCA> <http://rdfh.ch/0801/beol/-kF_VlCBQ6KTgUEYSAvfEg/values/28Mo6KbWRemI93GB0pgw5g> <http://rdfh.ch/0801/beol/3hGjlHmGSEmv4xksG-u9Rg/values/zq_D6-B4TF-mhYTT125Qig> <http://rdfh.ch/0801/beol/-y5jDcctRDy-6Ua9fe8fww/values/TxRLqvY0Sd2C_3cKuMVhNA> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/JtQC1WszRzy3qFdJhHqJ3A> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/7N_gmWSESxmCdsmGQonjUA> <http://rdfh.ch/0801/beol/0bUyZhemTt-Bdg93GJspjQ/values/WnPMqisITEixK-k_1sYYIg> <http://rdfh.ch/0801/beol/0EwJtpl-RVOANfhNE4lOdw/values/CJDRE1uNSdqS93XYE2uPUQ> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/MTA2NLkARHGnL49poTnnCw> <http://rdfh.ch/0801/beol/0VuEzASaT7S8t7w191oZVg/values/dKj_p6N_TryhpzlvOSDzCQ> <http://rdfh.ch/0801/beol/2kiTbDU-QSWIxN3tCcOKDg/values/0n6Zs4eER96-nKpWsqEOdQ> <http://rdfh.ch/0801/beol/1iT3P8KCQ9qSU4fWnwP3bg/values/d5SALRuOTR2FRebOGOA7eA> <http://rdfh.ch/0801/beol/2kiTbDU-QSWIxN3tCcOKDg/values/VkCF3KX0SqCZorIWjR7sbw> <http://rdfh.ch/0801/beol/1iT3P8KCQ9qSU4fWnwP3bg/values/F0bAh82OT6GjYwjt4QhOiQ> <http://rdfh.ch/0801/beol/2PeTcAP2R0uyhxKMR163pA/values/A7jizu3WReqfBsH5-lr4rw> <http://rdfh.ch/0801/beol/45jLFi29Rn-xl2wO8lCbkg/values/VQnWaOOVRYyxjfYrH5r0Ng> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/KQSKsJ96Ro6h2hKj9Bg7rA> <http://rdfh.ch/0801/beol/2bihBFCwSrmbLK6kTTyALQ/values/4GESJftCTtCHumi1lr1tRA> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/x230cFHYSHmE-W7_vXSmzw> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/ywenGo5uQLqG6Ap1EhWTqA> <http://rdfh.ch/0801/beol/-y5jDcctRDy-6Ua9fe8fww/values/B4KX2K8MTBmtfUwv4gu98A> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/1ozllxMJR3aSOcXBHWGWHg> <http://rdfh.ch/0801/beol/1ODu_YNhQKyUwr2HPn___w/values/Uv5hRdgtTJCmRWwlvAD-xA> <http://rdfh.ch/0801/beol/2bihBFCwSrmbLK6kTTyALQ/values/HHgYbn0qTXG9mu9uFpaj6w> <http://rdfh.ch/0801/beol/3hGjlHmGSEmv4xksG-u9Rg/values/0Hoczfn_RJCO4Ts7l8khNQ> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/XcDyt3prSZmBLU_HxKuQZQ> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/vCHSapKDTgGsv0M9Q8y8dA> <http://rdfh.ch/0801/beol/3tct2_3BRLmnSoA45OHBAA/values/vJrRiaGNSW6RgmIo2Br5JQ> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/OWX4R7SzRmqInZWbxlHo_A> <http://rdfh.ch/0801/beol/45jLFi29Rn-xl2wO8lCbkg/values/JjNZgeLaQPG0_nlHAOPa8g> <http://rdfh.ch/0801/beol/2bihBFCwSrmbLK6kTTyALQ/values/KYDm7E75SaSvbMZst4kLUA> <http://rdfh.ch/0801/beol/2kiTbDU-QSWIxN3tCcOKDg/values/6244B--ZTJGpOvGtBtpWGQ> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/10N5zTK_SLW_DUbOCHcF9w> <http://rdfh.ch/0801/beol/3tct2_3BRLmnSoA45OHBAA/values/mqMgJJ-aSj2ODSMbVuT5UA> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/_bZoz-M5T5uHmiIaHWZSow> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/px20BpBCRVC_8RAUzDDrdg> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/NqNyUSqSS2u27iMHTg-HjA> <http://rdfh.ch/0801/beol/2kiTbDU-QSWIxN3tCcOKDg/values/d0kx2cweRFWNKSs3uC7Wzw> <http://rdfh.ch/0801/beol/2bihBFCwSrmbLK6kTTyALQ/values/TZlz-8DUQxet7BTfuunYJA> <http://rdfh.ch/0801/beol/1UyZAGwCRra25xK5f8pTSg/values/kARq28WyTQuNS0dA2FEkTg> <http://rdfh.ch/0801/beol/-kF_VlCBQ6KTgUEYSAvfEg/values/5k7MFQjtQ6e3Zt2TVkl9HA> <http://rdfh.ch/0801/beol/05qZ_4zDRICkRoG5h4zoOQ/values/Gv3TBnvATTqlc6JKR-jQkA> <http://rdfh.ch/0801/beol/1ydw2I4nR1egWhm8OspBAg/values/f_LmajcBSmaxSzueLcPgjA> <http://rdfh.ch/0801/beol/1ydw2I4nR1egWhm8OspBAg/values/mOQJ0-6eQuOQLsC2akOOpg> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/Ijxfft75Qaau9u99_mMRhg> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/VShDWtA8TiKWqQHdd0qpMA> <http://rdfh.ch/0801/beol/45jLFi29Rn-xl2wO8lCbkg/values/32TcQXzVRT6AsUN1yxhhcQ> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/yhTgFLicSPegW1Y9eNRjXw> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/n0k5mgNVQPSi23RnYUmaUw> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/VefelQ2KTjmbHMTTKTD6sg> <http://rdfh.ch/0801/beol/2PeTcAP2R0uyhxKMR163pA/values/Mx_RUJYQQ-SEaB86g0Rp6g> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/ZCXeZVIYQZ-AoAk-43eMDw> <http://rdfh.ch/0801/beol/05qZ_4zDRICkRoG5h4zoOQ/values/h6So-5zhTDKyJV02dUAVSg> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/Bw1Dk4GwRwa9jvl50Wwq4Q> <http://rdfh.ch/0801/beol/-kF_VlCBQ6KTgUEYSAvfEg/values/q6KbEhHTQ_i8NrQBR3Q-YA> <http://rdfh.ch/0801/beol/0EwJtpl-RVOANfhNE4lOdw/values/Uzmrc-G1RX2qS8JK6H3TOg> <http://rdfh.ch/0801/beol/1UyZAGwCRra25xK5f8pTSg/values/PVhrOO4vT8un3Dr8pP2VvA> <http://rdfh.ch/0801/beol/0EwJtpl-RVOANfhNE4lOdw/values/aLy4snJZT16aLhDoLZfLRQ> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/9XS3TKbYSAGhxHHYnudssA> <http://rdfh.ch/0801/beol/2PeTcAP2R0uyhxKMR163pA/values/H3HT9cH_RE-g86jibxqdUQ> <http://rdfh.ch/0801/beol/2kiTbDU-QSWIxN3tCcOKDg/values/VCP6S5yMRAKlN3z4o0SEDg> <http://rdfh.ch/0801/beol/0_0Rs5IuT26hmcIGVu1NBA/values/7irnXvVBSA6B4SdXOY-7ow> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/pgQkZm1aTcikAcI5HYtERg> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/BqX2y-caRS2WXpWa4_IPYQ> <http://rdfh.ch/0801/beol/3tct2_3BRLmnSoA45OHBAA/values/io8YbHN_RuCNOZSXEreeMQ> <http://rdfh.ch/0801/beol/-0tI3HXgSSOeDtkf-SA00w/values/H4IZDQ1eS2WBrMhZRX-vkA> <http://rdfh.ch/0801/beol/1UyZAGwCRra25xK5f8pTSg/values/fvamE-RkS5i_f840TikfSA> <http://rdfh.ch/0801/beol/45jLFi29Rn-xl2wO8lCbkg/values/rnbrpUM2Q1iaSuNmCnmuLw> <http://rdfh.ch/0801/beol/20dsBNORRzuqx2AolMH_Sg/values/Njdq3nD4SxuaY7k3w63DDA> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/-pEgTyERReG206isJEJtLQ> <http://rdfh.ch/0801/beol/2bihBFCwSrmbLK6kTTyALQ/values/FCWcd4NiQX2EaSPm-XQMgw> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/x5WXFx6fTdSJtoa5MH7H6Q> <http://rdfh.ch/0801/beol/3tct2_3BRLmnSoA45OHBAA/values/Rz9LOthjQ7-U3IG41UkGeA> <http://rdfh.ch/0801/beol/2PeTcAP2R0uyhxKMR163pA/values/tNwzPY_cRpOgl-1fwvPezQ> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/lktHppa5RqyPmwVFRUBP4Q> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/Wkprd358Rma9REl0jKtToA> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/doxdoy6rSqOXIxJ6te3bXw> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/kW7c3dw-Rj6mb2sKytkYPQ> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/SJRySI6MTraQCMW7vTooOw> <http://rdfh.ch/0801/beol/0EwJtpl-RVOANfhNE4lOdw/values/iKg-VFviQdKT7xRaP72axQ> <http://rdfh.ch/0801/beol/-0tI3HXgSSOeDtkf-SA00w/values/MdjXlOTNRfqYKEWNIh6kfg> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/PKUrOieLSBysE2aXgPpA5Q> <http://rdfh.ch/0801/beol/2bihBFCwSrmbLK6kTTyALQ/values/dRdJh8djSZyDLcAkWA6Nsg> <http://rdfh.ch/0801/beol/2kiTbDU-QSWIxN3tCcOKDg/values/f7NhXm0eS9uyWZrTF4md5g> <http://rdfh.ch/0801/beol/-0tI3HXgSSOeDtkf-SA00w/values/SiEBL-EASQSNeFBprbjY3A> <http://rdfh.ch/0801/beol/-kF_VlCBQ6KTgUEYSAvfEg/values/iLteGtvsSOeXL_bZDNoVdw> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/LKuE2smyQa6QJ-kzUjEv-A> <http://rdfh.ch/0801/beol/0Srw9xbhRuKwSqpGmwhMKw/values/CK9iBb_WRO2YB1qYxbQo_Q> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/taW-kxP_RQ2Phw_4eJ_heg> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/wjAIl6kITD2cgiK5CnPuSA> <http://rdfh.ch/0801/beol/2bihBFCwSrmbLK6kTTyALQ/values/0fnxjaMdTjGXGI7CL-NtJA> <http://rdfh.ch/0801/beol/0_0Rs5IuT26hmcIGVu1NBA/values/FsfjsypWTL-qRz9lyvbUSA> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/5b_NqIM8RGS0-zvCGDIk7A> <http://rdfh.ch/0801/beol/-0tI3HXgSSOeDtkf-SA00w/values/JpGklfqZSxuu7VI1zIyucw> <http://rdfh.ch/0801/beol/2bihBFCwSrmbLK6kTTyALQ/values/Kd9hDnpXRi6l14jEFrTSiQ> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/pJxNtVEtSB-zbhPp4Aom_w> <http://rdfh.ch/0801/beol/20dsBNORRzuqx2AolMH_Sg/values/-oZMShYwSxy2ewacvHqZbQ> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/O8ZKgJMOTX-lugiCMil4_g> <http://rdfh.ch/0801/beol/45jLFi29Rn-xl2wO8lCbkg/values/oiEAAaXoTryPlr2se50Wcg> <http://rdfh.ch/0801/beol/45jLFi29Rn-xl2wO8lCbkg/values/skEd2S6ZRRee5Qik7EL4Og> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/9-lATMDJQqOdTu2odTZZUw> <http://rdfh.ch/0801/beol/1vDaL57JQHyfjTydRAmW_A/values/5OEnzIsXRFWWlgR5bkYX2w> <http://rdfh.ch/0801/beol/-y5jDcctRDy-6Ua9fe8fww/values/xoDD_qMWTVOl6Mbdbishzw> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/-ZSP4wHLS-GvgoQDscRrZg> <http://rdfh.ch/0801/beol/-kF_VlCBQ6KTgUEYSAvfEg/values/HSf2Lf2jTtGelR4zy3LSIg> <http://rdfh.ch/0801/beol/2PeTcAP2R0uyhxKMR163pA/values/bU959THbQmS5dz-P8YvYgA> <http://rdfh.ch/0801/beol/1iT3P8KCQ9qSU4fWnwP3bg/values/slcn-zU5RsO0Pu3a5VdE1A> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/utmWuFHwRSOZZ4R26ncJxA> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/0BQZFWOcTNuH3oqaPNtTqw> <http://rdfh.ch/0801/beol/0_0Rs5IuT26hmcIGVu1NBA/values/fE2zQTBHS_O7yH08FBjuQQ> <http://rdfh.ch/0801/beol/-y5jDcctRDy-6Ua9fe8fww/values/RMJirK-dR8CqBQavcz7lEw> <http://rdfh.ch/0801/beol/0_0Rs5IuT26hmcIGVu1NBA/values/SCzr4s99SgiZPAKePj-uaw> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/ERs4CNcjQRufYDeuyn9PjA> <http://rdfh.ch/0801/beol/-kF_VlCBQ6KTgUEYSAvfEg/values/0XnDPrYcQjqtUnOHmpkFCg> <http://rdfh.ch/0801/beol/2PeTcAP2R0uyhxKMR163pA/values/TVJgVanJRS2bR6ZJ6GOzMw> <http://rdfh.ch/0801/beol/0_0Rs5IuT26hmcIGVu1NBA/values/OnaYkzmKTXmEygrtWIDsww> <http://rdfh.ch/0801/beol/05qZ_4zDRICkRoG5h4zoOQ/values/PgYgjxi6TG6QW90B4EJqZg> <http://rdfh.ch/0801/beol/1iT3P8KCQ9qSU4fWnwP3bg/values/2EfIxCTVSEG73hsJYdXmww> <http://rdfh.ch/0801/beol/0_0Rs5IuT26hmcIGVu1NBA/values/siEPVzsERLajHyn8x_vkrA> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/4TXvKs5kRnmXM2Jw06w00g> <http://rdfh.ch/0801/beol/0_0Rs5IuT26hmcIGVu1NBA/values/6JLefDWsS9aG1YOgHzD6yg> <http://rdfh.ch/0801/beol/05qZ_4zDRICkRoG5h4zoOQ/values/Q1nnqbguR-qgpqxTvaLCSw> <http://rdfh.ch/0801/beol/1UyZAGwCRra25xK5f8pTSg/values/aM1Ap-G2SqCckpBX56D_rA> <http://rdfh.ch/0801/beol/20dsBNORRzuqx2AolMH_Sg/values/qtJblLZXTqu65_-Z1kxCuQ> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/7DLjc0EbTfWR27FslclL_A> <http://rdfh.ch/0801/beol/-kF_VlCBQ6KTgUEYSAvfEg/values/a6HVX11wScyYfPX8-6ZCOQ> <http://rdfh.ch/0801/beol/1ODu_YNhQKyUwr2HPn___w/values/UimY2UwOTPOGUIAbSgWwNQ> <http://rdfh.ch/0801/beol/45jLFi29Rn-xl2wO8lCbkg/values/reOkePFBT_2SIgwXYm11Sg> <http://rdfh.ch/0801/beol/0bUyZhemTt-Bdg93GJspjQ/values/RsxHJaTYSwC4p-_iYlQzEQ> <http://rdfh.ch/0801/beol/-y5jDcctRDy-6Ua9fe8fww/values/_35SrJ06Tj2A0DGrp3K1_g> <http://rdfh.ch/0801/beol/0EwJtpl-RVOANfhNE4lOdw/values/_SoSJvp2SIWbuOdFAbJhxQ> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/bCoNkiVhR5askrEdd2ARLA> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/giKsWTNHSl2_38AsiGyGcA> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/S-ZxGoaiRoeh-AoObij4QQ> <http://rdfh.ch/0801/beol/45jLFi29Rn-xl2wO8lCbkg/values/ROgm43QaSdCKJ6iyaYizyA> <http://rdfh.ch/0801/beol/2PeTcAP2R0uyhxKMR163pA/values/wpu1FsIDTHCvda-3r0yEqg> <http://rdfh.ch/0801/beol/-y5jDcctRDy-6Ua9fe8fww/values/1Rllte5ISbKUbjkxYivIYw> <http://rdfh.ch/0801/beol/-y5jDcctRDy-6Ua9fe8fww/values/huHXJNZGRZm5xPOPb1Z0WA> <http://rdfh.ch/0801/beol/0EwJtpl-RVOANfhNE4lOdw/values/gVr85VTmRz6t9de0fFRi2Q> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/fLtq-DidTcyqPJbOWGzz-w> <http://rdfh.ch/0801/beol/1ODu_YNhQKyUwr2HPn___w/values/V7xg7qIsRDmPe9n9L-i7vg> <http://rdfh.ch/0801/beol/-y5jDcctRDy-6Ua9fe8fww/values/pHt71EfiQWmBoQdW4AONQQ> <http://rdfh.ch/0801/beol/2bihBFCwSrmbLK6kTTyALQ/values/WcGtZKi1Qh2Io1CdBrdGww> <http://rdfh.ch/0801/beol/05qZ_4zDRICkRoG5h4zoOQ/values/DayNSrv7R6an8HZcLZA-Aw> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/yIO7rcfRQy6xxRfGQBETag> <http://rdfh.ch/0801/beol/0Srw9xbhRuKwSqpGmwhMKw/values/CidpHDYUS0KPIX7ctB47ig> <http://rdfh.ch/0801/beol/1vDaL57JQHyfjTydRAmW_A/values/dMxC6iXHSLmtHaxbebgZig> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/0eJYDFGhQKubCsT4syxHew> <http://rdfh.ch/0801/beol/0Srw9xbhRuKwSqpGmwhMKw/values/fySgOSCbR8GSogPoS6sdwA> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/OhDJUFahRXmz-Opm2GdOqA> <http://rdfh.ch/0801/beol/0_0Rs5IuT26hmcIGVu1NBA/values/NmDVv68YQwCC4O8LCX5HUw> <http://rdfh.ch/0801/beol/45jLFi29Rn-xl2wO8lCbkg/values/nh1uuWQlRZG11qDxzvJZSA> <http://rdfh.ch/0801/beol/-kF_VlCBQ6KTgUEYSAvfEg/values/sRwUmIRwRUaOiaajRqaqBQ> <http://rdfh.ch/0801/beol/-0tI3HXgSSOeDtkf-SA00w/values/_fCIiKjzRYqjXkEhykto5A> <http://rdfh.ch/0801/beol/1UyZAGwCRra25xK5f8pTSg/values/Z0pXX3I9S86ZZwTrd1QDgA> <http://rdfh.ch/0801/beol/3tct2_3BRLmnSoA45OHBAA/values/tp3PjHLaSj6BE-sYe3eLxQ> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/TapV8cKAR4KdESbbS6LO9Q> <http://rdfh.ch/0801/beol/3tct2_3BRLmnSoA45OHBAA/values/TAFDaJEtTj2jsi6zFvOGkA> }
 GRAPH <http://www.ontotext.com/explicit> {
     ?mainAndDependentResourceValueObject <http://www.knora.org/ontology/knora-base#valueHasStandoff> ?standoffNode .
 }
 GRAPH <http://www.ontotext.com/explicit> {
     ?standoffNode ?standoffProp ?standoffValue .
 }
 } UNION {
 VALUES ?mainAndDependentResourceValueObject { <http://rdfh.ch/0801/beol/1iT3P8KCQ9qSU4fWnwP3bg/values/MBpEl0J7S4C_erWObgzkUA> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/9GbjzXV0R_GmS36EjUT_fQ> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/u_K-he5BTcmKiRqOGWTVSg> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/_NazP9-mS52w0AQdH7D3mg> <http://rdfh.ch/0801/beol/0Srw9xbhRuKwSqpGmwhMKw/values/ml0tnFchRwqT87J6hc6R2A> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/y6VcU5D-Qzq8Tc2o-cAY7g> <http://rdfh.ch/0801/beol/0bUyZhemTt-Bdg93GJspjQ/values/hdK6ck0ORU6a5GouU4BqDQ> <http://rdfh.ch/0801/beol/-0tI3HXgSSOeDtkf-SA00w/values/rfao0tqnQTiYyLdlJrNF6w> <http://rdfh.ch/0801/beol/05qZ_4zDRICkRoG5h4zoOQ/values/uN-kDHqHRxC-AH_jpPcIkQ> <http://rdfh.ch/0801/beol/-y5jDcctRDy-6Ua9fe8fww/values/6-WLuWHIStm3UHUso9hS8Q> <http://rdfh.ch/0801/beol/3hGjlHmGSEmv4xksG-u9Rg/values/fpX5xC8dRVi9viOiJW1Q8A> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/asuHddlHT5W6rHBirMGRCA> <http://rdfh.ch/0801/beol/-kF_VlCBQ6KTgUEYSAvfEg/values/28Mo6KbWRemI93GB0pgw5g> <http://rdfh.ch/0801/beol/3hGjlHmGSEmv4xksG-u9Rg/values/zq_D6-B4TF-mhYTT125Qig> <http://rdfh.ch/0801/beol/-y5jDcctRDy-6Ua9fe8fww/values/TxRLqvY0Sd2C_3cKuMVhNA> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/JtQC1WszRzy3qFdJhHqJ3A> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/7N_gmWSESxmCdsmGQonjUA> <http://rdfh.ch/0801/beol/0bUyZhemTt-Bdg93GJspjQ/values/WnPMqisITEixK-k_1sYYIg> <http://rdfh.ch/0801/beol/0EwJtpl-RVOANfhNE4lOdw/values/CJDRE1uNSdqS93XYE2uPUQ> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/MTA2NLkARHGnL49poTnnCw> <http://rdfh.ch/0801/beol/0VuEzASaT7S8t7w191oZVg/values/dKj_p6N_TryhpzlvOSDzCQ> <http://rdfh.ch/0801/beol/2kiTbDU-QSWIxN3tCcOKDg/values/0n6Zs4eER96-nKpWsqEOdQ> <http://rdfh.ch/0801/beol/1iT3P8KCQ9qSU4fWnwP3bg/values/d5SALRuOTR2FRebOGOA7eA> <http://rdfh.ch/0801/beol/2kiTbDU-QSWIxN3tCcOKDg/values/VkCF3KX0SqCZorIWjR7sbw> <http://rdfh.ch/0801/beol/1iT3P8KCQ9qSU4fWnwP3bg/values/F0bAh82OT6GjYwjt4QhOiQ> <http://rdfh.ch/0801/beol/2PeTcAP2R0uyhxKMR163pA/values/A7jizu3WReqfBsH5-lr4rw> <http://rdfh.ch/0801/beol/45jLFi29Rn-xl2wO8lCbkg/values/VQnWaOOVRYyxjfYrH5r0Ng> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/KQSKsJ96Ro6h2hKj9Bg7rA> <http://rdfh.ch/0801/beol/2bihBFCwSrmbLK6kTTyALQ/values/4GESJftCTtCHumi1lr1tRA> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/x230cFHYSHmE-W7_vXSmzw> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/ywenGo5uQLqG6Ap1EhWTqA> <http://rdfh.ch/0801/beol/-y5jDcctRDy-6Ua9fe8fww/values/B4KX2K8MTBmtfUwv4gu98A> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/1ozllxMJR3aSOcXBHWGWHg> <http://rdfh.ch/0801/beol/1ODu_YNhQKyUwr2HPn___w/values/Uv5hRdgtTJCmRWwlvAD-xA> <http://rdfh.ch/0801/beol/2bihBFCwSrmbLK6kTTyALQ/values/HHgYbn0qTXG9mu9uFpaj6w> <http://rdfh.ch/0801/beol/3hGjlHmGSEmv4xksG-u9Rg/values/0Hoczfn_RJCO4Ts7l8khNQ> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/XcDyt3prSZmBLU_HxKuQZQ> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/vCHSapKDTgGsv0M9Q8y8dA> <http://rdfh.ch/0801/beol/3tct2_3BRLmnSoA45OHBAA/values/vJrRiaGNSW6RgmIo2Br5JQ> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/OWX4R7SzRmqInZWbxlHo_A> <http://rdfh.ch/0801/beol/45jLFi29Rn-xl2wO8lCbkg/values/JjNZgeLaQPG0_nlHAOPa8g> <http://rdfh.ch/0801/beol/2bihBFCwSrmbLK6kTTyALQ/values/KYDm7E75SaSvbMZst4kLUA> <http://rdfh.ch/0801/beol/2kiTbDU-QSWIxN3tCcOKDg/values/6244B--ZTJGpOvGtBtpWGQ> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/10N5zTK_SLW_DUbOCHcF9w> <http://rdfh.ch/0801/beol/3tct2_3BRLmnSoA45OHBAA/values/mqMgJJ-aSj2ODSMbVuT5UA> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/_bZoz-M5T5uHmiIaHWZSow> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/px20BpBCRVC_8RAUzDDrdg> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/NqNyUSqSS2u27iMHTg-HjA> <http://rdfh.ch/0801/beol/2kiTbDU-QSWIxN3tCcOKDg/values/d0kx2cweRFWNKSs3uC7Wzw> <http://rdfh.ch/0801/beol/2bihBFCwSrmbLK6kTTyALQ/values/TZlz-8DUQxet7BTfuunYJA> <http://rdfh.ch/0801/beol/1UyZAGwCRra25xK5f8pTSg/values/kARq28WyTQuNS0dA2FEkTg> <http://rdfh.ch/0801/beol/-kF_VlCBQ6KTgUEYSAvfEg/values/5k7MFQjtQ6e3Zt2TVkl9HA> <http://rdfh.ch/0801/beol/05qZ_4zDRICkRoG5h4zoOQ/values/Gv3TBnvATTqlc6JKR-jQkA> <http://rdfh.ch/0801/beol/1ydw2I4nR1egWhm8OspBAg/values/f_LmajcBSmaxSzueLcPgjA> <http://rdfh.ch/0801/beol/1ydw2I4nR1egWhm8OspBAg/values/mOQJ0-6eQuOQLsC2akOOpg> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/Ijxfft75Qaau9u99_mMRhg> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/VShDWtA8TiKWqQHdd0qpMA> <http://rdfh.ch/0801/beol/45jLFi29Rn-xl2wO8lCbkg/values/32TcQXzVRT6AsUN1yxhhcQ> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/yhTgFLicSPegW1Y9eNRjXw> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/n0k5mgNVQPSi23RnYUmaUw> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/VefelQ2KTjmbHMTTKTD6sg> <http://rdfh.ch/0801/beol/2PeTcAP2R0uyhxKMR163pA/values/Mx_RUJYQQ-SEaB86g0Rp6g> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/ZCXeZVIYQZ-AoAk-43eMDw> <http://rdfh.ch/0801/beol/05qZ_4zDRICkRoG5h4zoOQ/values/h6So-5zhTDKyJV02dUAVSg> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/Bw1Dk4GwRwa9jvl50Wwq4Q> <http://rdfh.ch/0801/beol/-kF_VlCBQ6KTgUEYSAvfEg/values/q6KbEhHTQ_i8NrQBR3Q-YA> <http://rdfh.ch/0801/beol/0EwJtpl-RVOANfhNE4lOdw/values/Uzmrc-G1RX2qS8JK6H3TOg> <http://rdfh.ch/0801/beol/1UyZAGwCRra25xK5f8pTSg/values/PVhrOO4vT8un3Dr8pP2VvA> <http://rdfh.ch/0801/beol/0EwJtpl-RVOANfhNE4lOdw/values/aLy4snJZT16aLhDoLZfLRQ> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/9XS3TKbYSAGhxHHYnudssA> <http://rdfh.ch/0801/beol/2PeTcAP2R0uyhxKMR163pA/values/H3HT9cH_RE-g86jibxqdUQ> <http://rdfh.ch/0801/beol/2kiTbDU-QSWIxN3tCcOKDg/values/VCP6S5yMRAKlN3z4o0SEDg> <http://rdfh.ch/0801/beol/0_0Rs5IuT26hmcIGVu1NBA/values/7irnXvVBSA6B4SdXOY-7ow> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/pgQkZm1aTcikAcI5HYtERg> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/BqX2y-caRS2WXpWa4_IPYQ> <http://rdfh.ch/0801/beol/3tct2_3BRLmnSoA45OHBAA/values/io8YbHN_RuCNOZSXEreeMQ> <http://rdfh.ch/0801/beol/-0tI3HXgSSOeDtkf-SA00w/values/H4IZDQ1eS2WBrMhZRX-vkA> <http://rdfh.ch/0801/beol/1UyZAGwCRra25xK5f8pTSg/values/fvamE-RkS5i_f840TikfSA> <http://rdfh.ch/0801/beol/45jLFi29Rn-xl2wO8lCbkg/values/rnbrpUM2Q1iaSuNmCnmuLw> <http://rdfh.ch/0801/beol/20dsBNORRzuqx2AolMH_Sg/values/Njdq3nD4SxuaY7k3w63DDA> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/-pEgTyERReG206isJEJtLQ> <http://rdfh.ch/0801/beol/2bihBFCwSrmbLK6kTTyALQ/values/FCWcd4NiQX2EaSPm-XQMgw> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/x5WXFx6fTdSJtoa5MH7H6Q> <http://rdfh.ch/0801/beol/3tct2_3BRLmnSoA45OHBAA/values/Rz9LOthjQ7-U3IG41UkGeA> <http://rdfh.ch/0801/beol/2PeTcAP2R0uyhxKMR163pA/values/tNwzPY_cRpOgl-1fwvPezQ> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/lktHppa5RqyPmwVFRUBP4Q> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/Wkprd358Rma9REl0jKtToA> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/doxdoy6rSqOXIxJ6te3bXw> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/kW7c3dw-Rj6mb2sKytkYPQ> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/SJRySI6MTraQCMW7vTooOw> <http://rdfh.ch/0801/beol/0EwJtpl-RVOANfhNE4lOdw/values/iKg-VFviQdKT7xRaP72axQ> <http://rdfh.ch/0801/beol/-0tI3HXgSSOeDtkf-SA00w/values/MdjXlOTNRfqYKEWNIh6kfg> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/PKUrOieLSBysE2aXgPpA5Q> <http://rdfh.ch/0801/beol/2bihBFCwSrmbLK6kTTyALQ/values/dRdJh8djSZyDLcAkWA6Nsg> <http://rdfh.ch/0801/beol/2kiTbDU-QSWIxN3tCcOKDg/values/f7NhXm0eS9uyWZrTF4md5g> <http://rdfh.ch/0801/beol/-0tI3HXgSSOeDtkf-SA00w/values/SiEBL-EASQSNeFBprbjY3A> <http://rdfh.ch/0801/beol/-kF_VlCBQ6KTgUEYSAvfEg/values/iLteGtvsSOeXL_bZDNoVdw> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/LKuE2smyQa6QJ-kzUjEv-A> <http://rdfh.ch/0801/beol/0Srw9xbhRuKwSqpGmwhMKw/values/CK9iBb_WRO2YB1qYxbQo_Q> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/taW-kxP_RQ2Phw_4eJ_heg> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/wjAIl6kITD2cgiK5CnPuSA> <http://rdfh.ch/0801/beol/2bihBFCwSrmbLK6kTTyALQ/values/0fnxjaMdTjGXGI7CL-NtJA> <http://rdfh.ch/0801/beol/0_0Rs5IuT26hmcIGVu1NBA/values/FsfjsypWTL-qRz9lyvbUSA> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/5b_NqIM8RGS0-zvCGDIk7A> <http://rdfh.ch/0801/beol/-0tI3HXgSSOeDtkf-SA00w/values/JpGklfqZSxuu7VI1zIyucw> <http://rdfh.ch/0801/beol/2bihBFCwSrmbLK6kTTyALQ/values/Kd9hDnpXRi6l14jEFrTSiQ> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/pJxNtVEtSB-zbhPp4Aom_w> <http://rdfh.ch/0801/beol/20dsBNORRzuqx2AolMH_Sg/values/-oZMShYwSxy2ewacvHqZbQ> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/O8ZKgJMOTX-lugiCMil4_g> <http://rdfh.ch/0801/beol/45jLFi29Rn-xl2wO8lCbkg/values/oiEAAaXoTryPlr2se50Wcg> <http://rdfh.ch/0801/beol/45jLFi29Rn-xl2wO8lCbkg/values/skEd2S6ZRRee5Qik7EL4Og> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/9-lATMDJQqOdTu2odTZZUw> <http://rdfh.ch/0801/beol/1vDaL57JQHyfjTydRAmW_A/values/5OEnzIsXRFWWlgR5bkYX2w> <http://rdfh.ch/0801/beol/-y5jDcctRDy-6Ua9fe8fww/values/xoDD_qMWTVOl6Mbdbishzw> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/-ZSP4wHLS-GvgoQDscRrZg> <http://rdfh.ch/0801/beol/-kF_VlCBQ6KTgUEYSAvfEg/values/HSf2Lf2jTtGelR4zy3LSIg> <http://rdfh.ch/0801/beol/2PeTcAP2R0uyhxKMR163pA/values/bU959THbQmS5dz-P8YvYgA> <http://rdfh.ch/0801/beol/1iT3P8KCQ9qSU4fWnwP3bg/values/slcn-zU5RsO0Pu3a5VdE1A> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/utmWuFHwRSOZZ4R26ncJxA> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/0BQZFWOcTNuH3oqaPNtTqw> <http://rdfh.ch/0801/beol/0_0Rs5IuT26hmcIGVu1NBA/values/fE2zQTBHS_O7yH08FBjuQQ> <http://rdfh.ch/0801/beol/-y5jDcctRDy-6Ua9fe8fww/values/RMJirK-dR8CqBQavcz7lEw> <http://rdfh.ch/0801/beol/0_0Rs5IuT26hmcIGVu1NBA/values/SCzr4s99SgiZPAKePj-uaw> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/ERs4CNcjQRufYDeuyn9PjA> <http://rdfh.ch/0801/beol/-kF_VlCBQ6KTgUEYSAvfEg/values/0XnDPrYcQjqtUnOHmpkFCg> <http://rdfh.ch/0801/beol/2PeTcAP2R0uyhxKMR163pA/values/TVJgVanJRS2bR6ZJ6GOzMw> <http://rdfh.ch/0801/beol/0_0Rs5IuT26hmcIGVu1NBA/values/OnaYkzmKTXmEygrtWIDsww> <http://rdfh.ch/0801/beol/05qZ_4zDRICkRoG5h4zoOQ/values/PgYgjxi6TG6QW90B4EJqZg> <http://rdfh.ch/0801/beol/1iT3P8KCQ9qSU4fWnwP3bg/values/2EfIxCTVSEG73hsJYdXmww> <http://rdfh.ch/0801/beol/0_0Rs5IuT26hmcIGVu1NBA/values/siEPVzsERLajHyn8x_vkrA> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/4TXvKs5kRnmXM2Jw06w00g> <http://rdfh.ch/0801/beol/0_0Rs5IuT26hmcIGVu1NBA/values/6JLefDWsS9aG1YOgHzD6yg> <http://rdfh.ch/0801/beol/05qZ_4zDRICkRoG5h4zoOQ/values/Q1nnqbguR-qgpqxTvaLCSw> <http://rdfh.ch/0801/beol/1UyZAGwCRra25xK5f8pTSg/values/aM1Ap-G2SqCckpBX56D_rA> <http://rdfh.ch/0801/beol/20dsBNORRzuqx2AolMH_Sg/values/qtJblLZXTqu65_-Z1kxCuQ> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/7DLjc0EbTfWR27FslclL_A> <http://rdfh.ch/0801/beol/-kF_VlCBQ6KTgUEYSAvfEg/values/a6HVX11wScyYfPX8-6ZCOQ> <http://rdfh.ch/0801/beol/1ODu_YNhQKyUwr2HPn___w/values/UimY2UwOTPOGUIAbSgWwNQ> <http://rdfh.ch/0801/beol/45jLFi29Rn-xl2wO8lCbkg/values/reOkePFBT_2SIgwXYm11Sg> <http://rdfh.ch/0801/beol/0bUyZhemTt-Bdg93GJspjQ/values/RsxHJaTYSwC4p-_iYlQzEQ> <http://rdfh.ch/0801/beol/-y5jDcctRDy-6Ua9fe8fww/values/_35SrJ06Tj2A0DGrp3K1_g> <http://rdfh.ch/0801/beol/0EwJtpl-RVOANfhNE4lOdw/values/_SoSJvp2SIWbuOdFAbJhxQ> <http://rdfh.ch/0801/beol/0newGctARC2jGUgunZVEmQ/values/bCoNkiVhR5askrEdd2ARLA> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/giKsWTNHSl2_38AsiGyGcA> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/S-ZxGoaiRoeh-AoObij4QQ> <http://rdfh.ch/0801/beol/45jLFi29Rn-xl2wO8lCbkg/values/ROgm43QaSdCKJ6iyaYizyA> <http://rdfh.ch/0801/beol/2PeTcAP2R0uyhxKMR163pA/values/wpu1FsIDTHCvda-3r0yEqg> <http://rdfh.ch/0801/beol/-y5jDcctRDy-6Ua9fe8fww/values/1Rllte5ISbKUbjkxYivIYw> <http://rdfh.ch/0801/beol/-y5jDcctRDy-6Ua9fe8fww/values/huHXJNZGRZm5xPOPb1Z0WA> <http://rdfh.ch/0801/beol/0EwJtpl-RVOANfhNE4lOdw/values/gVr85VTmRz6t9de0fFRi2Q> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/fLtq-DidTcyqPJbOWGzz-w> <http://rdfh.ch/0801/beol/1ODu_YNhQKyUwr2HPn___w/values/V7xg7qIsRDmPe9n9L-i7vg> <http://rdfh.ch/0801/beol/-y5jDcctRDy-6Ua9fe8fww/values/pHt71EfiQWmBoQdW4AONQQ> <http://rdfh.ch/0801/beol/2bihBFCwSrmbLK6kTTyALQ/values/WcGtZKi1Qh2Io1CdBrdGww> <http://rdfh.ch/0801/beol/05qZ_4zDRICkRoG5h4zoOQ/values/DayNSrv7R6an8HZcLZA-Aw> <http://rdfh.ch/0801/beol/-IS3VBv7QTGMwoFJXhYfNg/values/yIO7rcfRQy6xxRfGQBETag> <http://rdfh.ch/0801/beol/0Srw9xbhRuKwSqpGmwhMKw/values/CidpHDYUS0KPIX7ctB47ig> <http://rdfh.ch/0801/beol/1vDaL57JQHyfjTydRAmW_A/values/dMxC6iXHSLmtHaxbebgZig> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/0eJYDFGhQKubCsT4syxHew> <http://rdfh.ch/0801/beol/0Srw9xbhRuKwSqpGmwhMKw/values/fySgOSCbR8GSogPoS6sdwA> <http://rdfh.ch/0801/beol/1EfCDbaFQMqyiBp--TOj4g/values/OhDJUFahRXmz-Opm2GdOqA> <http://rdfh.ch/0801/beol/0_0Rs5IuT26hmcIGVu1NBA/values/NmDVv68YQwCC4O8LCX5HUw> <http://rdfh.ch/0801/beol/45jLFi29Rn-xl2wO8lCbkg/values/nh1uuWQlRZG11qDxzvJZSA> <http://rdfh.ch/0801/beol/-kF_VlCBQ6KTgUEYSAvfEg/values/sRwUmIRwRUaOiaajRqaqBQ> <http://rdfh.ch/0801/beol/-0tI3HXgSSOeDtkf-SA00w/values/_fCIiKjzRYqjXkEhykto5A> <http://rdfh.ch/0801/beol/1UyZAGwCRra25xK5f8pTSg/values/Z0pXX3I9S86ZZwTrd1QDgA> <http://rdfh.ch/0801/beol/3tct2_3BRLmnSoA45OHBAA/values/tp3PjHLaSj6BE-sYe3eLxQ> <http://rdfh.ch/0801/beol/0VE-1W0XRB-vzmZV2aemnA/values/TapV8cKAR4KdESbbS6LO9Q> <http://rdfh.ch/0801/beol/3tct2_3BRLmnSoA45OHBAA/values/TAFDaJEtTj2jsi6zFvOGkA> }
 GRAPH <http://www.ontotext.com/explicit> {
     ?mainAndDependentResourceValueObject <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.knora.org/ontology/knora-base#ListValue> .
 }
 GRAPH <http://www.ontotext.com/explicit> {
     ?mainAndDependentResourceValueObject <http://www.knora.org/ontology/knora-base#valueHasListNode> ?listNode .
 }
 GRAPH <http://www.ontotext.com/explicit> {
     ?listNode <http://www.w3.org/2000/01/rdf-schema#label> ?listNodeLabel .
 }
 }
 }
```

We are using the `Values` statement and just one variable representing a lot of IRIs. However,  you do not have to think about the fact that there are several instances of the same property when authoring the query.

Look at https://github.com/dhlab-basel/Knora/blob/develop/webapi/src/main/twirl/queries/sparql/v2/getResourcePropertiesAndValuesGraphDB.scala.txt for example.

There is a statement `?resource knora-base:hasValue ?valueObject .`. There could be many values or just one. Or even none.

Reply:
Is there a better way so we wouldn't have to repeat the same `Values` statement (it is quite long).

Reply:
So it gets all the list nodes in the main and dependent resources, even if the query didn't ask for all of them? And then in Scala you figure out what the user actually wants?

Reply:
I don't understand why you need this:

```
 GRAPH <http://www.ontotext.com/explicit> {
     ?mainAndDependentResourceValueObject <http://www.knora.org/ontology/knora-base#valueHasListNode> ?listNode .
 }
 GRAPH <http://www.ontotext.com/explicit> {
     ?listNode <http://www.w3.org/2000/01/rdf-schema#label> ?listNodeLabel .
 }
```

when you already have this:

```
     ?mainAndDependentResourceValueObject ?mainAndDependentResourceValueObjectProp ?mainAndDependentResourceValueObjectObj .
```

Isn't it redundant?

Reply:
It works like this that it gets all value objects that matched the criteria in the prequery (only the IRIs of those are contained in `VALUES`).

Please see the signature of `createMainQuery` https://github.com/dhlab-basel/Knora/blob/cb245958850143db089298872e0c0a3b2709bfc8/webapi/src/main/scala/org/knora/webapi/responders/v2/SearchResponderV2.scala#L1891

Reply:
> Isn't it redundant?

You get the value object that points to a list node (by its IRI). But you also want information about the list node itself.

Reply:
OK, why not unify the standoff and list node cases?

```
VALUES ?mainAndDependentResourceValueObject { ... }
 ?mainAndDependentResourceValueObject <http://www.knora.org/ontology/knora-base#valueHas> ?dependentObject .
 GRAPH <http://www.ontotext.com/explicit> {
     ?dependentObject ?dependentObjectProp ?dependentObjectPropObj .
 }
```

Reply:
> OK, why not unify the standoff and list node cases?

Good idea since in both cases there is an extra level of information to get. The question is how specific we should be (is it ok to get all the assertions about a list node).

I realize that I haven't really thought about permissions for list nodes.

Reply:
List nodes are small. Do they even have permissions? @subotic ?

Reply:
@benjamingeer Whatever we decide to do, it should be coordinated with https://github.com/dhlab-basel/Knora/blob/develop/webapi/src/main/twirl/queries/sparql/v2/getResourcePropertiesAndValuesGraphDB.scala.txt

Reply:
f0de5eb0c6fa4335a54397dd32ef747e02cbc959 puts the VALUE pattern for value objects in the WHERE clause's scope so it does not have to be repeated

Reply:
@benjamingeer Moving of the VALUE pattern into the WHERE clause's scope works. 

I think like this it should be fine.